### ISSUE TITLE
Show all downloaded episodes from all podcasts

### ISSUE THREAD

**Issue by hfel** [xgi/castero#94]
Hello! 

It would be nice if it was possible to view all the downloaded episodes from all podcasts in a single pane, à la podboat. I tend to download everything I want to listen to in advance as I don't have access to the internet at work where I listen to them, so a bird's eye view of the episodes I have available makes it much easier to decide, instead of having to scroll through individual podcasts one by one, especially if there are many of them.

**Comment by xgi** [comment]
Sure, I can add that! Probably this weekend.

**Comment by xgi** [comment]
Added with #96. Pressing 4 will go to the "downloaded perspective" which has a list of downloaded episodes. Note that I had to change the filename format for downloading episodes, so any of your episodes will need to be re-downloaded in order to be detected (sorry).

I haven't written tests for these changes yet so I'll leave this issue open until I do. I haven't found any problems, but let me know if you do. Thanks!
